swaps/rebalance.js

```diff
@@ -614,6 +615,7 @@ module.exports = (args, cbk) => {
           lnd: args.lnd,
           max_fee: defaultMaxFeeTotal,
           out_through: getOutbound.public_key,
```

Comment:
This should be replaced with `args.max_fee` ?

Reply:
if strict fee is specified i guess? I wrote this 3 years ago so i forget how this works

Reply:
I think it actually won't do anything as of today. There is a check near subscribeToProbe to see if it's strict then use max_fee but rebalance command never had a strict flag so its always undefined. I'll just replace it with `args.max_fee || 1337` for now.

Reply:
i think a default makes sense, could use the nullish coalescing operator here because if a max fee is set to zero you want to maintain that